javascript/extractor/src/com/semmle/jcorn/CustomParser.java

```diff
@@ -196,8 +196,12 @@ protected Node parseLetExpression(Position startLoc, boolean maybeStatement) {
 		this.expect(TokenType.parenR);
 
 		if (this.type == TokenType.braceL) {
-			if (!maybeStatement)
-				this.unexpected();
+			if (!maybeStatement) {
```

Comment:
I can not find anything about let expressions on TC39, are they an E4X thing? If so, don't we need a e4x feature check around here?

Reply:
They are a historic Mozilla-specific extension. We already had support for it, but this PR improves it. Both invocations of this method are guarded by `options.mozExtensions()` checks further up.